### DISCUSSION TITLE
Plugin - DisablesSSLVerification

### DISCUSSION THREAD

**Discussion by jdc1898** [saloonphp/saloon#295] [General]
I know it is discouraged however, there are certain circumstances that disabling SSL checks are required. In case anyone needs a plugin in v2, here is how I did it.

```
use Saloon\Contracts\PendingRequest;

trait DisablesSSLVerification
{
    /**
     * Disable SSL verification on requests. I hope you know this is bad.
     *
     * @param \Saloon\Contracts\PendingRequest $pendingRequest
     * @return void
     */
    public function bootDisablesSSLVerification(PendingRequest $pendingRequest): void
    {
        $pendingRequest->config()->merge([
            'verify' => false,
        ]);
    }
}
```

**Comment by Sammyjo20** [comment]
Hey, @jdc1898 thank you very much for posting this - it can definitely be useful for the community. I'm going to convert this into a discussion so it can be visible in the future.
